src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -346,11 +404,47 @@ class ConsignmentRouteSpec extends AnyFlatSpec with Matchers with TestRequest wi
     result.getString(field)
   }
 
-  private def createSeries(bodyId: UUID): Unit = {
-    val sql = "INSERT INTO Series (SeriesId, BodyId) VALUES (?,?)"
-    val ps: PreparedStatement = databaseConnection.prepareStatement(sql)
-    ps.setString(1, "6e3b76c4-1745-4467-8ac5-b4dd736e1b3e")
-    ps.setString(2, bodyId.toString)
-    ps.executeUpdate()
+  private def setUpConsignments(consignmentParams: List[(UUID, String, List[UUID])]): Unit = {
```

Comment:
Might be worth making a case class? It's easier to follow than a tuple because the parameters have names rather than just positions.